### ISSUE TITLE
Decide/Handle the fate of "standard" brim query lib

### ISSUE THREAD

**Issue by mason-fish** [brimdata/brim#2233]
Since the app is moving in a more generic direction, the current local query lib we ship brim with seems like it no longer fully represent the use cases we support. Do we want to nest those in a special `security` sub-folder in the lib, and then also include another sub-folder with more generic data? Or perhaps an `examples` folder to accompany the docs?

**Comment by mason-fish** [comment]
We've decided that the fate of the current security-centric "standard" brim lib is to be moved into the Brimcap repo as an importable json file instead. When Brim v1.0 ships, it will no longer provide any of it's own set of queries, and we will include in the release notes a description of where this particular folder went (which has been hacked to be read only) and how to get it back. Getting it back will require downloading the .json file from the brimcap repo and then dragging it into the app.

In the future, this brimcap query lib importing process can be automated further as an external plugin. That is, a plugin or extension which the user can browse to and install all through the app, similar to how many IDEs do.

**Comment by mason-fish** [comment]
re-opening because we may want to keep this lib for users that already have it

**Comment by philrz** [comment]
Once this is complete, I plan to look at making some more additions to the query library as described in https://github.com/brimdata/brimcap/issues/267.

**Comment by philrz** [comment]
I can't recall if we've discussed allowing these to be loaded over the network, e.g., pasting/dragging a URL as an alternative to loading a local JSON file. In addition to saving the user the effort of downloading the query JSON file in a separate step, while reviewing old issues I was reminded of https://github.com/brimdata/brim/issues/1421 and https://github.com/brimdata/brim/issues/1422, which both could be made possible once loading over the network is possible. In the case of the latter, I could imagine the app using URL-accessible metadata (e.g., last-modified time) to check if the original has changed in the time since it was first loaded and prompt the user to ask if they want to refresh.

**Comment by philrz** [comment]
I've verified that as of Brim commit 7f07d1e2 the former "stock" Query Library entries can now be downloaded from their new home in the Brimcap repo as mentioned in the [Brimcap README](https://github.com/brimdata/brimcap#brimcap-queries), then imported and applied. I happened to use [this test pcap from the Zeek team](https://github.com/401trg/detections/blob/master/pcaps/20171220_smb_net_user.pcap) along with the Windows Network Activity in the test, which verifies #2431.

https://user-images.githubusercontent.com/5934157/187290049-0d61357c-2eb8-4f71-9215-ad52c32e25b9.mp4

The workflow here does point to a few of our open issues that could help smooth this out further:

1. #2503 would save the extra step of downloading before importing
2. These Query Library entries lack the "FROM" pin that's now part of saved queries in the new Session Flow. Issues #2500, #2478, and #2477 all discuss ways we might make it easier for users to make this transition.

**Comment by philrz** [comment]
I've also verified the bullet above:

> * Make sure the lib gets removed for existing users upgrading to Zui

In the attached video, we begin with a GA Brim `v0.31.0` install that has the traditional "Brim" folder of Zeek/Suricata queries as well as an additional custom query. We then quit and upgrade to a test draft Zui `v1.0.0` artifact (based on Brim commit a9c4fef, with artifacts built in [this Actions run](https://github.com/brimdata/brim/actions/runs/2951664828)) and find the "Brim" folder is gone as expected, but the custom query is still intact.

https://user-images.githubusercontent.com/5934157/187315193-a81a993b-d28d-4502-876e-1fbe79b189af.mp4

I've also added a note-to-self in #2393 to make sure we mention all this in the Release Notes for Zui v1.0.0.

Also, regarding the point above about adding tests to avoid a repeat of what we saw with #2431, I've opened new issue #2504.

That seems to verify all the angles for this issue, so, closing. Thanks @mason-fish!
